hiredis.c

```diff
@@ -516,7 +516,7 @@ int redisvFormatCommand(char **target, const char *format, va_list ap) {
 
     hi_free(curargv);
     *target = cmd;
-    return totlen;
+    return (int)totlen;
```

Comment:
These commands really should return a `ssize_t` result to be consistent with the use of `size_t`, but I didn't want to modify their signature since they are a public API